letter.md

```diff
@@ -1,25 +1,21 @@
 
-This combination of features allows users to trust that any software they install from CRAN is likely to work, be compatible with other CRAN packages, and leave their local R package library in a consistent state. The great effort undertaken to move these software package pain points out of the way of statistics practitioners has made a large contribution toward R becoming an extremely popular statistical programming environment.
+Because of its success, CRAN has become a fundamental global resource, not only for independent researchers, but also for large organizations across academia, government, and industry, all of whom all depend on the ability to contribute packages in a reliable and predictable manner. The scope and the need have grown bigger than one individual or small team, and it is appropriate that the R community exercise a voice with respect to the conduct and administration of CRAN.
 
-Unfortunately, we, the undersigned R community members, are unified in our concern that CRAN is showing signs that it is struggling to create a consistent experience for contributors. 
+We, the undersigned R community members, are unified in concern that CRAN is struggling to provide a consistent experience for contributors.
 
-In particular there appears to be pattern of increasing and shifting the burden of maintaining packages on CRAN via regular introduction of new criteria and policies, coupled with an uptick in heavy-handed enforcement practices. Some of these new constraints are not well documented, causing much confusion that is adding to the burden on both contributors and CRAN administrators.
+New criteria and policies are sometimes introduced without enough notice or documentation for contributors to follow, and enforcement practices can be discourteous and severe. Contributors are experiencing ambiguity, disruption, and frustration, and they are changing their behaviour to reduce exposure to CRAN's policies and checks (e.g. skipping all tests on CRAN). This not only undermines CRAN's checking system but also the R language itself since CRAN checks are used to validate core changes.
 
-This pattern is driving behavioural change in contributors among us. Many are acting to reduce their exposure to CRAN's policies and checks. This not only undermines CRAN's checking system but also the R language itself since CRAN checks are used to validate core changes.
+To reverse these trends, we are petitioning CRAN for reform. We request measures that:
 
-There are also contributors for whom the increasing burden has reached a tipping point, leading them to feel frustrated and discouraged. There is evidence that this frustration is mirrored by some CRAN administrators with numerous accounts of communications that are apparently coloured by it. 
```

Comment:
I'd like to keep this as well. I put this is in for all the developers who feel discouraged by interactions with CRAN that have disappointed them. The experience isn't universal, but I feel it is important to acknowledge, since it shouldn't be happening at all.

Reply:
I do see the need for emphasis, but as a fresh reader, that paragraph felt like it added repetition and drama that did not serve the core message. And mentioning numerous accounts of communication leaves us with more explaining to do because we are also claiming there is not enough communication. I reworked the paragraph above it to include some of the details and sentiment.